Content.Shared/Movement/Components/MovementSpeedModifierComponent.cs

```diff
@@ -22,8 +22,8 @@ public sealed partial class MovementSpeedModifierComponent : Component
         public const float DefaultFriction = 20f;
         public const float DefaultFrictionNoInput = 20f;
 
-        public const float DefaultBaseWalkSpeed = 2.5f;
-        public const float DefaultBaseSprintSpeed = 4.5f;
+        public const float DefaultBaseWalkSpeed = 3f;
```

Comment:
This is maybe not good? I'm not sure, it made a lot of sense on Park, but if this is an optional change should we really be altering the default speed on a Component level?